### ISSUE TITLE
CMake-based pure C library on Flatpak

### ISSUE THREAD

**Issue by silvioprog** [flatpak/flatpak#2748]
Hello Flatpak team.

I would like to use Flatpak to publish a small project I'm developing, however, I have a question. Can I publish a CMake-based pure C library at Flathub? If so, [the library](https://risoflora.github.io/libsagui) is a cross-platform library which helps to develop web servers or frameworks.

Thank you!

**Comment by kirbyfan64** [comment]
No, in general anyone using your library will put the relevant module in their Flatpak manifest. If you want to make it easier, you can add an example module to your documentation.

**Comment by silvioprog** [comment]
> No, in general anyone using your library will put the relevant module in their Flatpak manifest.

Interesting. After your message I found the section [Manifests](http://docs.flatpak.org/en/latest/manifests.html) at Flatpak page and it will help me a lot.

I've used Launchpad for years, but I'm inclined to retire it and migrate definitively my applications to Flatpak.

> If you want to make it easier, you can add an example module to your documentation.

I'm going to try it and share the results on social networks.

Thanks a lot for helping! :)
